lib/trino-parquet/src/main/java/io/trino/parquet/reader/ColumnReaderFactory.java

```diff
@@ -108,10 +109,12 @@ private static List<ColumnReaderProvider> getColumnReaderProviders(PrimitiveType
                     LongFlatColumnReader.doubleProvider(),
                     createProvider(DoubleColumnReader::new));
             case BINARY -> ImmutableList.of(
+                    Int128FlatColumnReader.longDecimalReaderProvider(),
```

Comment:
nit:static import

Reply:
It feels useful to keep the "Int128" prefix as it makes it easier to see what type of Block the output will be